src/DomBuilder/BaseDomBuilder.php

```diff
@@ -84,6 +95,18 @@ protected function createElement(string $name, ?string $value = null): \DOMEleme
         }
     }
 
+    public function removeElement(string $node): void
```

Comment:
Hi, thanks for you contribution. Can you explain the use of this method ? I don't see any reference to it and would suggest to remove it.